packages/grid/helm/syft/values.yaml

```
@@ -112,6 +112,7 @@ seaweedfs:
 # =================================================================================
 
 proxy:
+  enabled: false
```

```suggestion
  enabled: true
```
Currently in a few use-cases , we use the internal traefik proxy.
By default it would be good to have it enabled.
So it is easy for new users trying out the system.
For customized deployments we could disable it,  by modifying values.yaml during deployment.

Also, In upcoming iterations, we are trying to remove the dependence on the internal proxy and remove it, to have the routing at the ingress controller itself.